From 4d5df34b2bba33b718981aad96ca810c4f0f9e2f Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 25 Jun 2020 19:58:48 +0000
Subject: [PATCH 0943/1000] MANTIS 0057761 Darken Screen using the
 Navigation-Pad add PlatformIF::switchEnableDisplay add
 ComponentIF::switchEnableDevice add DataHandling::DH_DELAY_DATA,
 DataHandling::DH_SUPPRESS_DELAYED_DATA

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1576 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index 49084b13..13a09183 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -162,6 +162,7 @@ module PlatformManagement
         
         code enableDevice();
         code disableDevice();
+        code switchEnableDevice();
     };
 
     enum DataStatusExtension
@@ -324,7 +325,9 @@ module PlatformManagement
     enum DataHandling
     {
       DH_FORWARD_DATA_TO_APP,
-      DH_SUPPRESS_DATA
+      DH_SUPPRESS_DATA,
+      DH_DELAY_DATA,
+      DH_SUPPRESS_DELAYED_DATA
     };
 
     enum CounterType
@@ -430,6 +433,7 @@ module PlatformManagement
         
         code enableDisplay();
         code disableDisplay();
+        code switchEnableDisplay();
     };
 };
 
-- 
2.41.0.windows.1

